buildSrc/src/main/java/Application.kt

```diff
@@ -1,8 +1,8 @@
 object Application {
     const val applicationId = "com.fueled.technicalchallenge"
-    const val compileSdk = 31
-    const val minSdk = 23
-    const val targetSdk = 31
+    const val compileSdk = 34
+    const val minSdk = 27
```

Comment:
👀 any particular reason to bump the min sdk?

Reply:
no particular reason, thought I'd bump it as well since this is just an exercise

Reply:
Let's retain the minsdk since it would be better to be consistent with what we use on our projects